hazelcast/src/main/java/com/hazelcast/spi/merge/MergingView.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.spi.merge;
+
+/**
+ * Marker interface representing a read-only view of a data structure for the merging process after a split-brain.
```

Comment:
minor: using shorter column width makes javaDoc easy to read for me, as in above copyright text.

Reply:
Done.